CONTRIBUTING.md

```diff
@@ -51,6 +51,79 @@ For source code integrity , Hyperledger Bevel GitHub pull requests are accepted
 8. Work on the feedbacks to revise the PR if there are any comments
 9. If not, the PR gets approved , delete feature branch post the merge
 
+## Create local branch
+
+> Whenever you begin work on a new feature or bugfix, it's important that you create a new branch.
+1. Clone your fork to your local machine
+
+2. Setup your local fork to keep up-to-date (optional)
+   ```
+   # Add 'upstream' repo to list of remotes
+   git remote add upstream https://github.com/hyperledger/bevel.git
+   # Verify the new remote named 'upstream'
+   git remote -v
+   # Checkout your main branch and rebase to upstream.
+   # Run those commands whenever you want to synchronize with main branch
+   git fetch upstream
+   git checkout main
+   git rebase upstream/main
+   ```
+3. Create your branch.
+   ```
+   # Checkout the main branch - you want your new branch to come from main
+   git checkout main
+   # Create a new branch named `<newfeature>` (give simple informative name)
+   git branch <newfeature>
+   ```
+4. Checkout your branch and add/modify files.
+   ```
+   git checkout <newfeature>
+   git rebase main
+   # Happy coding !
+   ```
+5. Commit changes to your branch.
+   ```
+   # Commit and push your changes to your fork
+   git add -A
+   git commit -s -m "<type>[optional scope]: <description>"
+   git push origin <newfeature>
+   ```
+6. Once you've committed and pushed all of your changes to GitHub, go to the page for your fork on GitHub, select your development branch, and click the pull request button.
+
+7. Repeat step 3 to 6 when you need to prepare posting new pull request.
+
+NOTE: Once you submitted pull request to Bevel repository, step 6 is not necessary when you made further changes with `git commit --amend` since your amends will be sent automatically.
+
+NOTE: You can refer original tutorial ['GitHub Standard Fork & Pull Request Workflow'](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+
+### Directory structure
+
+Whenever you begin to use your codes on Hyperledger Bevel, you should follow the directory strecture on Hyperledger Bevel.
+The current directory structure is described as the following:
+
+
+> - automation/ : This folder contains the Input configuration file and Jenkinsfile for r3-corda, hyperledger fabric which helps in automation of corda/fabric network.
```

Comment:
The automation folder structure description is totally wrong. Please check again what the Jenkins file contain.